### PR TITLE
BugTracker Fix Build Warnings

### PR DESCRIPTION
**Pull request by kingthorin**
Building anything in the Alpha branch causes a bunch of warnings due to
deprecation etc. This PR addresses the deprecated items in the
bugtracker addon.

### CODE REVIEW

**Comment by kingthorin** [comment]
Do we want manifest updates for this or just leave it to be included in the next release?

**Comment by thc202** [comment]
Yes, the version number should be updated (if needed), not worth mentioning this type of changes though.

File: src/org/zaproxy/zap/extension/bugtracker/DialogAddBugzillaConfig.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ protected boolean validateFields() {
     
     @Override
     protected void performAction() {
-        config = new BugTrackerBugzillaConfigParams(getNameTextField().getText(), getPasswordTextField().getText(), getBugzillaUrlTextField().getText());
+        config = new BugTrackerBugzillaConfigParams(getNameTextField().getText(), getPasswordTextField().getPassword().toString(), getBugzillaUrlTextField().getText());
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/bugtracker/DialogAddBugzillaConfig.java]: This change does not yield the same result, it needs to created a new String but better resolution of the deprecation is to use the char array.

File: src/org/zaproxy/zap/extension/bugtracker/DialogAddGithubConfig.java [hunk 2]
```diff
@@ -134,7 +134,7 @@ protected boolean validateFields() {
     
     @Override
     protected void performAction() {
-        config = new BugTrackerGithubConfigParams(getNameTextField().getText(), getPasswordTextField().getText(), getRepoUrlTextField().getText());
+        config = new BugTrackerGithubConfigParams(getNameTextField().getText(), getPasswordTextField().getPassword().toString(), getRepoUrlTextField().getText());
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/bugtracker/DialogAddGithubConfig.java]: Same comment.

File: src/org/zaproxy/zap/extension/bugtracker/DialogAddBugzillaConfig.java (hunk 1 above)
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/bugtracker/DialogAddBugzillaConfig.java]: K

**Comment by kingthorin** [comment]
This addon is already at the next manifest version and does not appear to have been released yet.

**Comment by thc202** [comment]
Thank you!

**Pull request merged**